nuntium/tests/api/message_resource_test.py

```diff
@@ -318,3 +318,36 @@ def test_including_a_non_email_in_the_author_email(self):
 
         self.assertEquals(response.status_code, 400)
         self.assertFalse(Message.objects.filter(author_name='Felipipoo'))
+
+    def test_get_list_of_messages_filtered_by_popolo_uri(self):
+        popolo_uri = 'http://example.com/popolo.json#person-123'
+        # Create a person with a popolo identifier
+        person = PopoloPerson.objects.create(name='Test')
+        person.identifiers.create(scheme='popolo_uri', identifier=popolo_uri)
+        # Create an extra message that we don't expect to see in the response
+        message = Message.objects.create(
+            content=u'Test Content',
+            author_name=u'Felipe',
+            author_email=u"falvarez@votainteligente.cl",
+            subject=u'Fiera es una perra feroz',
+            writeitinstance=self.writeitinstance,
+            persons=[person],
+            )
+        Confirmation.objects.create(message=message)
+        message.recently_confirmated()
+
+        # Fetch a filtered list of message from the API
+        url = '/api/v1/message/'
+        params = self.data.copy()
+        params.update({
+            'person__popolo_uri': popolo_uri,
+        })
```

Comment:
A tiny thing, but I'd put this on one line instead of three

Reply:
👍